omero/developers/Python.rst

```diff
@@ -375,6 +431,28 @@ Read-Annotate groups, this will include other users' data - see
     image = conn.getObject("Image", imageId)
     print "Image: ", image,
 
+- **To save an Object's owner as another user (Admins only)**
+
+::
+
+    tag_ann = omero.gateway.TagAnnotationWrapper(conn)
+    tag_ann.setTextValue("Not owned by me")
+    # update details of the wrapped omero.model.AnnotationI _obj
+    tag_ann._obj.details.owner = ExperimenterI(userId, False)
+    tag_ann.save()
+
+    # Alternatively, we can connect as another user, to perform several tasks
+    user = conn.getObject("Experimenter", userId).getName()
+    user_conn = conn.suConn(user)
+    # This annotation will be owned by user
+    map_ann = omero.gateway.MapAnnotationWrapper(user_conn)
+    map_ann.setNs(namespace)
+    map_ann.setValue(key_values)
+    map_ann.save()
+    # Link will be owned by the user
```

Comment:
is "the user" here different from the "user" above ?

Reply:
yes. exactly as ```# This annotation will be owned by user``` above.